File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        printf("Height: ");
        height = get_int();
    } while (height < 0 || height > 23);

    for (int i = 0; i < height; i++)
    {
        int empty_space = height - i;
        for (int j = 0; j < empty_space; j++)
        {
            printf(" ");
        }

        printf("#");

        for (int z = 0; z < i; z++)
        {
            printf("#");
        }

        printf("\n");
    }


}

File: pset6/mario.py
import cs50

def main():
    # request pyramid height
    while(True):
        print("Please input height of the pyramid: ", end="")
        input_value = cs50.get_int()
        if (input_value > 0 and input_value < 23):
            break

    # print pyramid
    for i in range(1, input_value + 1):
        # print empty space
        print(" " * (input_value - i), end="")
        # print pyramid level
        print("#" * i)

if __name__ == "__main__":
    main()
